src/stylesheets/components/_key-factors.scss

```diff
@@ -0,0 +1,291 @@
+/*----------- Key Factors -----------*/
+
+.key-factors {
+  color: $florence-gray;
+  margin-top: 150px;
+
+  @media ($screen-sm) {
+    margin-top: 65px;
+  }
+
+  h3 {
+    color: $florence-orange;
+    font-size: 1em;
+    margin-bottom: -8px;
+    text-transform: uppercase;
+  }
+}
+
+.section-title {
+  color: $florence-blue;
+  font-size: 0.65em;
+  letter-spacing: 2px;
+  text-align: center;
+  text-transform: uppercase;
+  width: 100%;
+}
+
+.section-intro {
+  color: $florence-blue;
+  font-family: $secondary-font;
+  font-size: 1.7em;
+  line-height: 1.6em;
+  margin: auto;
+  text-align: center;
+  width: 57vw;
+
+  @media ($screen-sm) {
+    font-size: 1.4em;
+    width: 85vw;
+  }
+}
+
+.factor-bg {
+  background-image: url('../images/body-burst.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 1000px auto;
+  height: 740px;
+  margin-top: -110px;
+  width: 100%;
+
+  @media ($screen-sm) {
+    background-image: url('../images/body-burst-mobile.png');
+    background-position: top;
+    background-size: 100vw;
+    margin-top: -190px;
+    min-height: 1400px;
+  }
+
+  @media ($screen-xs) {
```

Comment:
in this case, it's best to go `xs` first and then define `sm` afterwards. if you define it out of order, as the case here, it makes it hard to read. on top of this, and this may be totally made up, but `xs` takes precedence over `sm` so anything you are trying to do in the `sm` breakpoint may be unset by the `xs` styles that are defined later on the document.